database/things/fortran.pldb

```diff
@@ -2,6 +2,7 @@ title Fortran
 appeared 1957
 type pl
 creators John Backus
+spec https://fortran-lang.org/en/
```

Comment:
This links seems to be more of a general description of the language, rather than a specification.